understanding/20/bypass-blocks.html

```diff
@@ -11,12 +11,12 @@ <h1>Understanding Bypass Blocks</h1>
    <section id="brief">
 		<h2>In brief</h2>
 		<dl>
-		<dt>Objective</dt><dd>Users can more easily navigate by keyboard</dd>
-		<dt>Author task</dt><dd>Provide a means of skipping repeating content</dd>
-		
-		</dl>
-		
-		</section>
+         <dt>Goal</dt><dd>Users can more easily navigate by keyboard.</dd>
+         <dt>What to do</dt><dd>Provide a means of skipping repeating content.</dd>
+         <dt>Why it's important</dt><dd>Users reliant on the keyboard can move around pages efficiently.</dd> 
```

Comment:
@rainbreaw commented:
> In understanding/20/bypass-blocks.html, understanding/20/focus-order.html, understanding/20/focus-visible.html and understanding/20/keyboard-no-exception.html:

> Consider extending why it's important to include individuals who use switch devices or other AT inputs.

Reply:
@GreggVan identified a number of SCs where he also felt it useful to explicitly differentiate between people who rely on the keyboard interface and those using a keyboard.

IMO, there is questionable value adding that into the short description. I believe the essential concept to 'get' is that it's for people who don't just point and click. Also, the concept of the keyboard interface is elaborated on in the Understanding docs (to greater or less extend). It's another situation where I think a generic explanation of switches and other devices that use the keyboard interace could be made as a PR that was added to a number of these keyboard-linked SCs.

In this particular instance, where we've already said "by keyboard" explicitly in the goal, I'm going to suggest the why can be a bit broader without confusing anyone. How about this?
```suggestion
         <dt>Why it's important</dt><dd>Users reliant on the keyboard interface can move around pages efficiently.</dd> 
```